src/pages/help-centre/ib-programme.tsx

```diff
@@ -16,34 +17,43 @@ const StyledListItem = styled.li<{ marginTop: string }>`
     margin-top: ${(props) => props.marginTop};
 `
 
-const SignUp = ({ text }: ArticleProps) => (
-    <ArticleWrapper>
-        <StyledHeader as="h4">{text}</StyledHeader>
-        <Text>
-            <Localize
-                translate_text="To become an IB, you'll need to be an existing affiliate with a Deriv account and an MT5 Synthetic real account. If you do, you can apply to become an IB by writing to us at <0>affiliates@deriv.com</0>. <1>Get more info about our IB programme</1>."
-                components={[
-                    <ExternalLink
-                        to={'mailto:affiliates@deriv.com'}
-                        target="_blank"
-                        external
-                        weight="bold"
-                        rel="noopener noreferrer"
-                        key={0}
-                    />,
-                    <ExternalLink
-                        to={'https://deriv.com/partners/affiliate-ib/'}
-                        external
-                        target="_blank"
-                        rel="noopener noreferrer"
-                        key={1}
-                    />,
-                ]}
-            />
-        </Text>
-    </ArticleWrapper>
-)
+const ExternalLink = styled(LocalizedLinkText)`
+    text-decoration: none;
+    font-size: var(--text-size-s);
+    font-weight: normal;
+    color: var(--color-red);
+
+    :hover {
+        text-decoration: underline;
+    }
 
+    @media ${device.tabletL} {
+        font-size: 16px;
+    }
+`
+const SignUp = ({ text }: ArticleProps) => {
+    return (
+        <ArticleWrapper>
+            <StyledHeader as="h4">{text}</StyledHeader>
+            <Text>
+                <Localize
+                    translate_text="First, you need to be an affiliate with a Deriv account and an MT5 Derived real account. Then, <0></0> to apply for an IB account. <1>Get more info about our IB programme.</1>"
+                    components={[
+                        <LiveChatLinkText text="contact us via live chat" key={0} />,
```

Comment:
```suggestion
    <LiveChatLinkText text="_t_contact us via live chat_t_" key={0}/>
```
since you used `text` directly in `LiveChatLinkText`, you have to wrap the text with `_t_`.